<!DOCTYPE html>
<html>

<head>
    <title>Tesla's Diary</title>
    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>
    <link rel="stylesheet" href="../../../css/tufte.css" />
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});
    </script>
    <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>
</head>

<body>
    <header>
        <nav class="group">
            <a href="http://www.teslasdiary.com"><h2 style="text-align:left;float:left;">Tesla's Diary</h2></a>
            <a href="http://www.teslasdiary.com/about.html"><h2 style="text-align:right;float:right; margin-right:20%" class='no-print'>About</h2></a>
        </nav>
    </header>
    <br>
    
    <h1>DNA-Level Splice Junction Prediction using Deep Recurrent Neural Networks. </h1>
    <p class="subtitle">By Byunghan Lee, Taehoon Lee, Byunggook Na, Sungroh Yoon</p>
    <content class="mn-demo">
        <p>
        <p>A eukaryotic gene consists of multiple exons (protein coding regions) and
introns (non-coding regions), and a splice junction refers to the boundary
between a pair of exon and intron. Precise identification of spice junctions on
a gene is important for deciphering its primary structure, function, and
interaction. Experimental techniques for determining exon/intron boundaries
include RNA-seq, which is often accompanied by computational approaches.
Canonical splicing signals are known, but computational junction prediction
still remains challenging because of a large number of false positives and
other complications. In this paper, we exploit deep recurrent neural networks
(RNNs) to model DNA sequences and to detect splice junctions thereon. We test
various RNN units and architectures including long short-term memory units,
gated recurrent units, and recently proposed iRNN for in-depth design space
exploration. According to our experimental results, the proposed approach
significantly outperforms not only conventional machine learning-based methods
but also a recent state-of-the-art deep belief network-based technique in terms
of prediction accuracy.
</p>
<p><a href="http://arxiv.org">Donate to arXiv</a></p></p>
        
        <p><a href="http://www.arxiv.org">Donate to arxiv.org</a></p>
        <p><a target="_blank" href="http://arxiv.org/pdf/1512.05135" class='no-print'>Download</a>
        </p>
        <p><a target="_blank" href="http://arxiv.org/abs/1512.05135">Original</a>
        </p>
    </content>
    <br>
    <br>
    <br>
</body>

</html>